assets/constants.ts

```diff
@@ -5893,5 +5893,8 @@ export default {
   ],
 
   "DEV_MODE": true,
-  "EMULATOR_MODE": true
+  "EMULATOR_MODE": true,
+  "ASSET_TYPE_AUDIO": "audio",
```

Comment:
Do you still think we need to handle these constants in this PR or [PR 13159](https://github.com/oppia/oppia/pull/13159) ? 
There are going to be 4 PRs in total dependent on these changes. I am thinking it is better to handle this in a separate PR. 
It can get confusing for the reviewers as well.

Reply:
If https://github.com/oppia/oppia/pull/13159 gets merged by tomorrow, we will resolve the conflict else will create a separate PR.